packit_service/utils.py

```diff
@@ -70,7 +70,12 @@ def get_package_nvrs(built_packages: List[dict]) -> List[str]:
         if package["arch"] == "src":
             continue
 
-        epoch = f"{package['epoch']}:" if package["epoch"] != 0 else ""
+        epoch = (
+            f"{package['epoch']}:"
+            if package["epoch"] is not None and package["epoch"] != 0
```

Comment:
```suggestion
            if package["epoch"]
```

But I would probably add a comment…